R/shift_configuration.R

```diff
@@ -27,7 +27,8 @@
 #' \item{Y}{input trait vector/matrix.}
 #' \item{tree}{input tree.}
 #' \item{shift.configuration}{estimated shift positions, i.e. vector of indices of edges where the estimated shifts occur.}
-#' \item{shift.values}{estimates of the shift values.}
```

Comment:
This is dangerous: you changed the names of the output from `shift.values` to `shift.optima`. For users this is bad: creates non-compatibility or earlier code with newer package version. You should change this back to `shift.values`, if you have not done so already, in later commits. I have not had time to check.